### PR TITLE
[WIP] Adds actions for updating and numbering agenda items

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Agenda_item.numbering: Corresponding to [OpenSlides-action-crew/openslides-backend-coordination#74](https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/74)
Agenda_item.update: Corresponding to [OpenSlides-action-crew/openslides-backend-coordination#76](https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/76)

### CODE REVIEW

File: client/src/app/core/repositories/agenda/agenda-item-repository.service.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
                 'item_number',
                 'comment',
                 'closed',
-                'type',
+                'agenda_type',
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/agenda/agenda-item-repository.service.ts:44]: This is wrong: It is `type`, see the models.txt.

File: client/src/app/core/repositories/agenda/agenda-item-repository.service.ts [hunk 2]
```diff
@@ -120,11 +121,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
      * @param viewModel the view model that the update is based on
      */
     public async update(update: Partial<AgendaItem>, viewModel: ViewAgendaItem): Promise<void> {
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/agenda/agenda-item-repository.service.ts:123]: Please just delete this method and ose the one from the base class.

File: client/src/app/shared/models/agenda/agenda-item.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ export class AgendaItem extends BaseModel<AgendaItem> {
     public item_number: string;
     public comment: string;
     public closed: boolean;
-    public type: number;
+    public agenda_type: number;
```
Comment (FinnStutzenstein) [inline client/src/app/shared/models/agenda/agenda-item.ts:27]: Please undo the changes.

**Comment by GabrielInTheWorld** [comment]
Duplicate of #41
